modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/consensus/block/Validator.scala

```diff
@@ -73,7 +74,37 @@ object Validator {
             if (!enoughTips) "Not enough tips" else "",
             if (!enoughTxs) "No transactions" else ""
           ).filter(_.nonEmpty).mkString(", ")
-          logger.debug(s"Cannot start own consensus: ${reason}")
+          logger.debug(s"Cannot start currency own consensus: $reason")
+        }
+    } yield res
+
+  def canStartOwnDataConsensus[F[_]: Async](
```

Comment:
I think it's defined in wrong place. `consensus/block` is for the currency consensus and it's defined in `dag-l1`.
![image](https://github.com/Constellation-Labs/tessellation/assets/4959931/bf741847-4316-40af-8a97-a17a2b261c06)

DAG L1 knows nothing about data updates. The fn you've just created should be defined in the currency L1. There is a namespace `org.tessellation.currency.l1.domain.dataApplication.consensus`

Reply:
Also name "ownCurrencyConsensus" sounds strange in dag-l1. DAG is technically a currency but we rather use currency for non-dag currencies. Wdyt?

Reply:
I agree